index.html

```diff
@@ -85,6 +85,37 @@ <h2>Start your spec!</h2>
       The <dfn>bar</dfn> attribute.
     </p>
   </section>
+  <section id="RTCAudioReceiverStats-dict*">
+    <h3>
+      <dfn>RTCAudioReceiverStats</dfn> dictionary non-standardized members
+    </h3>
+    <pre class="idl">partial dictionary RTCAudioReceiverStats {
+      unsigned long long jitterBufferFlushes;
+      unsigned long long delayedPacketOutageSamples;
+};</pre>
+    <dl data-link-for="RTCAudioReceiverStats" data-dfn-for="RTCAudioReceiverStats" class=
+    "dictionary-members">
+      <dt>
+        <dfn><code>jitterBufferFlushes</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long-long">unsigned long long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Counts the total number of times the jitter buffer
+          reaches its maximum capacity and gets flushed.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>delayedPacketOutageSamples</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long-long">unsigned long long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Measures the number of audio samples that were concealed due to delayed packets. This gives an indication of how well the jitter buffer's network adaptation is working.
```

Comment:
This sounds like the same thing as https://w3c.github.io/webrtc-stats/#dom-rtcaudioreceiverstats-concealedsamples. Is there a difference between a concealed packet due to packet loss and concealed packet due to a delayed packet? What's a "delayed packet"?

Reply:
concealedSamples includes both samples that are concealed due to packet loss and due to delayed packets (which causes jitter buffer underflow).